src/ICeWall/doc/add_pad.md

```diff
@@ -0,0 +1,51 @@
+## add_pad
+
+### Synopsis
+```
+  % add_pad [-name <name>] \
+                    [-type <type>] \
+                    [-cell <library_cell>] \
+                    [-signal <signal_name>] \
```

Comment:
`place_cell` appears to be the answer when there is no signal connected to the cell (ESD, etc), but since that doesn't follow the same command interface as the rest of ICeWall (and fill gets added over cells inserted using `place_cell`).
It would be great to be able to use `add_pad` without a signal. Currently generating ERROR-0122

Reply:
The -signal option is not mandatory for add_pad, when the -signal option is defined, it is used to select the instance in the input netlist that will be placed. 
If there is a cell in the input netlist that needs to be placed in the padring, then using the -inst_name option should allow the correct instance to be specified.
If there is not a cell in the input netlist corresponding to the required placement, then ICeWall will create an instance in any of the following cases:
1. The cell is declared to by "physical_only"
2. The cell is declared to be a control cell (not available through the TCL API)
3. The option to create padcells in the design has been set (not available from the TCL API)
4. The cell is for a power or ground net
Otherwise we report ERROR-0122. 

The example creates breaker cells in this manner, specifying -inst_name, but not -signal, so I expect that for most applications (e.g. an ESD cell to be added to the ring) I think it would be reasonable to define the cell as "physical_only"

ICeWall is expecting to have full control over everything that gets added into the padring, and does not account for blockages created by calls to place_cell. Cells added by palce_cell are expected to be within the area used by padring.

Reply:
Marking the cell physical_only did the trick, maybe it would make sense to add that to the documentation for add_pad and define_pad_cell to indicate that ESD cell and others that do not connect should be physical only and that when using the add_pad without -signal is should connect to a physical_only cell or something? Nothing major, just clarification.